### DISCUSSION TITLE
W5550 How to set fixed IP ?

### DISCUSSION THREAD

**Discussion by obdevel** [earlephilhower/arduino-pico#2640] [Q&A] [answered]
This hardware (W5500-EBV-Pico2) works fine with a DHCP-assigned IP address. I can't find how to set a fixed IP address. The Wiznet5500lwIP class doesn't seem to have the setLocalIP() method or a constructor that takes these args (per the Arduino Ethernet class docs).

Thanks.

**Comment by JAndrassy** [comment, marked as answer]
use `config` as with WiFii or use EthernetCompat.h for the standard Ethernet '`begin`

**Comment by obdevel** [comment]
Thanks. eth->config(ip, dns, gateway, subnet); worked fine
